Makefile.sdk.mk

```diff
@@ -271,8 +302,6 @@ endef
 
 $(eval $(call make-git-meson-module-rules,zlib,build/fs-%/lib/pkgconfig/zlib.pc,))
 
-$(eval $(call make-git-autotools-module-rules,xz,build/fs-%/lib/pkgconfig/liblzma.pc,))
```

Comment:
Can't we pass in the patch as an argument like we do for the other modules that require patches?

Reply:
Sorry. I think the examples of this were in Makefile.toolchain.mk, since I hadn't touch those, I hadn't seen it. I have copied and adapted the necessary fragment now and it looks much better. Thanks.